app/api/v1/endpoints/login.rb

```diff
@@ -17,16 +15,11 @@ class Login < Grape::API
         post :login do
           user = User.find_by_username(params[:username].downcase)
```

Comment:
The console error is happening because user is nil, because cannot find username in the db.

Try the same line in the console to make sure user is not nil